CONTRIBUTING.md

```diff
@@ -42,9 +42,14 @@ bundle exec rake bin/rails db:migrate RAILS_ENV=test
 ### Run the specs
 ```
 bundle exec rake (all specs)
+bundle exec rspec (all specs)
 bundle exec rspec <path-to-file> (single file)
 ```
 
+## Running rubocop
+```
+bundle exec rubocop -a <path-to-file> (single file)
```

Comment:
Including the `-a` flag will run the rubocop autocorrect. This could potentially be surprising to someone unfamiliar with rubocop running it for the first time. I think we should either remove the `-a` flag from this documentation or include some kind of explanation / warning

Reply:
added the explanation above.